Adjust English in README (#172)
home-assistant-libs/ha-ffmpeg: README.md

- Analyse a video/audio stream for noise or motion detection
- Grab image from a stream

> Be careful that you protect function calls to this library with `asyncio.shield`.

Reviewer: The English of this sentence is still not correct. I'll check with some people if we know the intention of this sentence.

Reviewer: ```suggestion
Take care to protect function calls to this library with `asyncio.shield`.
```

Author: Yes, the better way to phrase that.
